retriever/lib/datapackage.py

```diff
@@ -164,6 +164,8 @@ def create_json():
     contents['retriever_minimum_version'] = "2.0.dev"
     contents['encoding'] = clean_input("encoding: ", ignore_empty = True)
     if is_empty(clean_input("encoding: ", ignore_empty = True)) : contents['encoding'] = ENCODING
+    contents['dataset_availability'] = clean_input("Message for non-availability of dataset: ", ignore_empty = True)
+    if is_empty(clean_input("Message for dataset availability: ", ignore_empty = True)) : contents['dataset_availability'] = 'True'
```

Comment:
1. How about checking  contents dictionary for key 'dataset_availability' and if we don't have a value in that dictionary, we can set it to  contents['dataset_availability'] = 'True'

2. I think this alone` contents['dataset_availability'] = clean_input("Message for non-availability of dataset: ", ignore_empty = True)` would be enough. In this case we are saying if for some case we create a and want to say that the data is having a problem right now. we could go ahead and include it else just ignore it


The same issue happens with 
```
contents['encoding'] = clean_input("encoding: ", ignore_empty = True)
if is_empty(clean_input("encoding: ", ignore_empty = True)) : contents['encoding'] = ENCODING
```
When you run this program, try to make the first variable empty when supplying encoding. 
It will ask for` encoding` twice. 
So the first time you are asked for the value and you don't supply one.  `contents['encoding'] = clean_input("encoding: ", ignore_empty = True)` the next if statement will display the same message again.
I think we need to fix that.